### ISSUE TITLE
Exception raised using rx_didChangeAuthorizationStatus

### ISSUE THREAD

**Issue by kjf** [ReactiveX/RxSwift#89]
I'm just wrapping my head around the whole Rx thing so apologies if this is something of my own doing.

My viewDidLoad method looks something like this:

```
  override func viewDidLoad() {
    super.viewDidLoad()

    locationManager.rx_didChangeAuthorizationStatus
      >- subscribeNext() { status in
        println(status)
      }
      >- disposeBag.addDisposable
  }
```

Upon execution an exception is raised inside the rx_didChangeAuthorizationStatus CLLocationManager extension.

```
Could not cast value of type '__NSCFNumber' (0x7fe523b6b0b8) to 'C.CLAuthorizationStatus' (0x102105250).
```

Using swift 1.2 and iOS SDK 8.4

**Comment by kzaher** [comment]
Hi @kjf,

I'm looking at the code, and I think you are right, this looks like a bug. 

We are casting to `AnyObject as! CLAuthorizationStatus` and I think compiler will need our help there.

`AnyObject -> Int -> CLAuthorizationStatus`

**Comment by kzaher** [comment]
Hi,
the fix should be in develop branch right now

commit c4b392c86d75719ded24037a717a6eb8a0ba51db
Author: Krunoslav Zaher <krunoslav.zaher@gmail.com>
Date:   Wed Aug 12 19:37:07 2015 +0200

    Fix for #89 (CLAuthorizationStatus casting crash)

I've tried to confirm does it work, let me know if it doesn't :)

**Comment by kjf** [comment]
Just tested and it's working great for me. Thanks!

**Comment by kzaher** [comment]
I'm preparing a small patch release end week. So it will be released then ;)

Let me know if you find anything else.

**Comment by kjf** [comment]
Awesome, will do.

**Comment by kzaher** [comment]
I've released 1.9.1

This should be fixed there :)
